website/docs/patterns/filter-patterns/partials/specifications/specifications.md

```diff
@@ -0,0 +1,139 @@
+
+### Spacing
+
+In the case of multiple Segmented Groups within a filter bar, use a `16px` gap between each group.
+
+![Multiple groups in a filter bar](/assets/patterns/filter-patterns/filter-bar-anatomy-02.png)
+
+Sometimes a set of global filter functions is necessary to refresh the data set or surface an action to reveal additional overflow filters in a Flyout. In this case the global functions should be aligned to the end of the container with space between the primary filters and the global functions.
+
+![Functions within a filter bar](/assets/patterns/filter-patterns/filter-bar-anatomy-03.png)
+
+## Filter sidebar
+
+With more real estate in a filter sidebar, the list that was responsible for displaying the filter values in the filter bar can be expanded to display all applicable values at once in a group.
+
+![Sidebar anatomy](/assets/patterns/filter-patterns/sidebar-anatomy.png =350x*)
+
+| Element | Usage |
+|---------|-------|
+| Sidebar container | Wraps the filters or multiple groups in a vertical orientation. |
+| Divider | (Optional, but recommended) Increases visual differentiation between each group. |
+| [Checkbox Group](/components/form/checkbox) | Displays the values in a checkbox group, supporting the selection of multiple items. |
+| [Radio Group](/components/form/radio) | Displays the values in a radio group, supporting the selection of a single value. |
+
+### Spacing
+
+Spacing within a filter sidebar extends the guidelines for spacing within a [form pattern](/patterns/form-patterns).
+
+- If a divider separates the groups use a `24px` vertical gap on the top and bottom of the divider.
+- If no divider is used, groups should receive a `32px` gap between them.
+
+**Ex 3.0: Spacing in a sidebar that uses dividers**
+
+![Sidebar spacing](/assets/patterns/filter-patterns/sidebar-spacing-divider.png =350x*)
+
+**Ex 3.1: Spacing in a sidebar that doesn't use dividers**
+
+![Sidebar spacing without a divider](/assets/patterns/filter-patterns/sidebar-spacing-no-divider.png =350x*)
+
+### Filters within a Flyout
+
+If its necessary for filters to overflow into a [Flyout](/components/flyout), use the same organizational method and spacing guidelines as the filter sidebar.
+
+![Filters within a Flyout](/assets/patterns/filter-patterns/flyout-filter-spacing.png)
+
+## Applied filters
+
+![Applied filters anatomy](/assets/patterns/filter-patterns/applied-filters-anatomy.png)
+
+| Element | Usage |
+|---------|-------|
+| Applied Filter Container | Wraps the label and applied filters to enforce consistent spacing |
+| Label | Labels the applied filters and communicates whether filters have been applied or not. |
+| Filters | A list of dismissible [Tags](/components/tag) that communicate what filters have been applied to the dataset. |
+| Bulk clear | Clear all applied filters with a single bulk action. Uses a small [Tertiary button](/components/button) with an icon. |
+
+### Spacing
```

Comment:
This content will be removed, if we introduce a `TagList` element, right? (in that case, these specs are already implemented by the components itself)

Reply:
I'd keep this in until we publish the `TagList` – can make a note to review it at that point.